src/web/src/pages/layouts/MDXPageBase.tsx

```diff
@@ -23,17 +22,17 @@ const useStyles = makeStyles((theme) => {
       '& h1': {
         color: theme.palette.text.secondary,
         fontSize: 24,
-        paddingLeft: '5px',
+        padding: '0vw 0vw 0vw 10vw',
       },
       '& h2': {
         color: theme.palette.text.secondary,
         fontSize: 20,
-        paddingLeft: '5px',
+        padding: '2vw 5vw 0vw 10vw',
```

Comment:
I don't understand using vertical units for the width too (`vh` vs `vw`).  Screens aren't square.